### DISCUSSION TITLE
🚀 Feature Request: Support load_extensions (for spellfix1)

### DISCUSSION THREAD

**Discussion by omrikurtz** [cloudflare/workers-sdk#13496] [Feature Requests]
### Describe the solution

Currently, we cannot use D1 since we need the usage of editdist3 (levenshtein distance) which is included in the spellfix1 extension.

However, when trying to run `SELECT load_extension('spellfix1.so', 'sqlite3_init_extension')`, I am presented with `not authorized` error.

**Comment by thelovekesh** [comment]
@penalosa will you add this extension in D1?
